functests/0_config/config.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
```

Comment:
unused?

Reply:
used by Reject()

Reply:
Fair enough. And BTW this is exacty why I dislike the dot imports.